app/services/auth.js

```diff
@@ -11,15 +13,25 @@ export default class AuthService extends Service {
    */
   @service router;
 
+  @tracked currentUser = null;
+
   /**
    *
    * @param {string} userId
    */
-  loginWithUserId(userId) {
+  async loginWithUserId(userId) {
```

Comment:
Doesn't use `await` in this method (no need for async), seems odd to add it when it isn't needed?

Reply:
Good catch. There should be a `await this.loadCurrentUser()` in there